### ISSUE TITLE
`<Pointout />` 's Focus indicator color on Close button lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#568]
![image](https://user-images.githubusercontent.com/3180185/159586740-92a650c4-28a6-47e2-8d47-d7668ceaf615.png)

<dl><dt>Description</dt><dd> The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
In Dark theme -
Focus indicator on Close button lacks contrast ratio of 3:1.
With Foreground : #FFFFFF and Background: #F7846C contrast ratio is 2.5:1 </dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.</dd><dt>Source code</dt><dd><pre><code><button class="Pointout__dismiss" type="button" aria-label="Close" tabindex="-1" data-focusable="true"><div aria-hidden="true" class="Icon Icon--close"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></button></code></pre></dd><dt>Checkpoint</dt><dd>Non-Text Contrast - States of User Interface Components (1.4.11.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Pointout</dd></dl>

**Comment by scurker** [comment]
This component is getting deprecated/removed, see #900
